src/apps/vod/M1Message.msg

```diff
@@ -11,7 +11,7 @@ cplusplus {{
 #include "inet/networklayer/common/L3Address.h"
 }};
 
-struct inet::L3Address;
+class inet::L3Address { @existingClass; }
```

Comment:
You could simply use an import here, I guess.

Reply:
Yes, you are right. Was changed to an import.